ts/components/menu/ConversationListItemContextMenu.tsx

```diff
@@ -128,15 +129,12 @@ export const PinConversationMenuItem = (): JSX.Element | null => {
   const showPinUnpin = useShowPinUnpin(conversationId);
```

Comment:
same comment, `const showPinUnpin = useShowPinUnpin(conversationId);` should be removed.
and `if (!showPinUnpin) {` should be `if (!togglePinConversation) {`